crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1316,21 +1322,52 @@ def parallel_upgrade_compute_nodes(compute_nodes)
             "Error while upgrading compute nodes. " + e.message
           )
         end
-        # FIXME: should we paralelize this as well?
+        # reboot block (reboot could be just initiated)
```

Comment:
What do you mean with "reboot could be just initiated" ?

Reply:
I just mean that we do not need to wait for the node to be back online (as it seems to be confusing, I should reword it)